src/data/services/ZendeskApiService.js

```diff
@@ -0,0 +1,25 @@
+import apiClient from '../apiClient';
+import { configuration } from '../../config';
+
+class ZendeskApiService {
+    static baseUrl = configuration.LMS_BASE_URL;
+    static createZendeskTicket(options) {
+      const url = `${ZendeskApiService.baseUrl}/zendesk_proxy/v1`;
+      const data = {
+        requester: {
+          email: options.emailAddress,
+          name: options.enterpriseName,
+        },
+        subject: 'Enterprise Admin Support',
```

Comment:
Should we allow admins to customize the subject (through a new form field maybe)? I think it will get confusing for ECS if they have a number of tickets that all have the same subject

Reply:
Not saying we have to do it, but something to check with ECS / Joe / @brittneyexline about

Reply:
yea, that seems reasonable to me as well, to let subject be a field the user enters.